Doc/library/sqlite3.rst

```diff
@@ -693,7 +690,7 @@ Connection objects
       :meth:`~Cursor.executescript` on it with the given *sql_script*.
       Return the new cursor object.
 
-   .. method:: create_function(name, narg, func, *, deterministic=False)
+   .. method:: create_function(name, narg, func, /, *, deterministic=False)
```

Comment:
Did we forget to deprecate keyword use for this API?

Reply:
Sorry, I do not understand.

Reply:
For the other APIs, we deprecated keyword use in order to warn users about these breaking changes. For this API, I don't think we went through the deprecation process.

Reply:
It was deprecated for the first three parameters. `deterministic` is keyword-only boolean parameter, and it should be.